docs/guide_running_portal_locally.md

```diff
@@ -0,0 +1,61 @@
+# Guide to running portal locally with a custom portal config
+
+This guide shows you how to test a portal config (aka `gitops.json` file) locally, so that you don't have to wait until the new `gitops.json` file is uploaded to GitHub and merged to see if your changes will break everything!
+
+Assuming you want to edit the `gitops.json` file for `preprod.healdata.org`:
```

Comment:
I think it makes sense to keep heal or another real commons as the example because of the references to the `cdis-manifest` folder -- following these instructions to the letter will work with preprod.healdata.org but won't work with my-commons.org.